articles/azure-stack/user/azure-stack-version-profiles-azurecli2.md

```diff
@@ -153,38 +152,34 @@ Use the following steps to connect to Azure Stack:
     d. To register the user in an AD FS environment, use:
 
       ```azurecli
-      az cloud register \
-        -n AzureStack  \
+      az cloud register -n AzureStack  \
         --endpoint-resource-manager "https://management.local.azurestack.external" \
         --suffix-storage-endpoint "local.azurestack.external" \
         --suffix-keyvault-dns ".vault.local.azurestack.external"\
-        --endpoint-active-directory-resource-id "https://management.adfs.azurestack.local/<tenantID>" \
-        --endpoint-active-directory-graph-resource-id "https://graph.local.azurestack.external/"\
-        --endpoint-active-directory "https://adfs.local.azurestack.external/adfs/"\
         --endpoint-vm-image-alias-doc <URI of the document which contains virtual machine image aliases> \
         --profile "2018-03-01-hybrid"
       ```
+    >[!NOTE]  
+    > The parameter endpoint-vm-image-alias-doc is optional. The operator sets up a URL similar to https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-compute/quickstart-templates/aliases.json for azurestack images and provides this URL to its users. If this url is not provided only functionality missing would be auto completion of available images with the command `az vm image`. The user can get the available images by doing `az vm image list`
```

Comment:
I don't think this parameter is optional. Vm creation uses this endpoint as a reference to build image url. If this is not specified, vm creation will fail. Please check if using whole image urn works before publishing this.

Reply:
Specifying the whole image urn as `--image` parameter works for Vm creation if the vm-image-alias endpoint isn't specified. Please make sure to add a note to specify the whole image Urn (Eg. MicrosoftWindowsServer:WindowsServer:2012-R2-Datacenter:10.0.0) in case the above endpoint isn't specified. Also `az vm image list` uses vm-image-alias-endpoint to provide available images. The user will have to use `az vm image list --all` to list all the images available from server.